Vagrantfile

```diff
@@ -52,13 +52,19 @@ Vagrant.configure("2") do |config|
       node.vm.hostname = "node-#{i}"
       node_ip = NODE_IP_NW + "#{10 + i}"
       node.vm.network :private_network, ip: node_ip
+      node.vm.network "private_network", virtualbox__intnet: "swp1", auto_config: false
 
       # Add disk for gluster client
       node.vm.provider "virtualbox" do |vb|
         unless File.exist?("disk-#{i}.vmdk")
           vb.customize ["createhd", "--filename", "disk-#{i}.vmdk", "--size", DISK_SIZE]
         end
         vb.customize ['storageattach', :id,  '--storagectl', 'SCSI', '--port', 2, '--device', 0, '--type', 'hdd', '--medium', "disk-#{i}.vmdk"]
+        vb.customize ["modifyvm", :id, "--cpus", 2]
+        vb.customize ["modifyvm", :id, "--memory", 4096]
```

Comment:
line 29-32 has a set of vm configuration about cpus and memory. Remove the old one?